uber/site_sections/departments.py

```diff
@@ -0,0 +1,218 @@
+from uber.common import *
+
+
+@all_renderable(c.PEOPLE)
+class Root:
+
+    def index(self, session, message=''):
+        departments = session.query(Department).order_by(Department.name).all()
+        return {
+            'message': message,
+            'departments': departments
+        }
+
+    @requires_dept_admin
+    def form(self, session, message='', **params):
+        department_id = params.get('id')
+        if not department_id or department_id == 'None':
+            raise HTTPRedirect('index')
```

Comment:
The `not department_id` is exactly as you've described. The `'None'` check is defensive programming (paranoia) on my part 😅 

In many places in our codebase we set the model id to `"None"` in our `form.html` templates. But departments use `new.html` to create new departments instead of `form`. Essentially, I didn't want someone to accidentally (or maliciously) POST an id with a string value of `"None"`.

Probably just late night programming paranoia.